### ISSUE TITLE
Dedimania plugin not working [TMF]

### ISSUE THREAD

**Issue by 3zet** [EvoEsports/minicontrol#27]
Just as in the picture below: 
![dedimania](https://github.com/EvoEsports/minicontrol/assets/69508064/a4050f32-7e8f-4b34-a1b9-97cdbafa5276)
I tried to login by using both the server account password and a community code.

**Comment by reaby** [comment]
join discord: https://discord.com/invite/evoesports

**Comment by reaby** [comment]
short answer: no it doesn't work atm, due bun has bug with its http stack.

so, i've been trying to fix this issue some 3 months already, tried a lot of different things.
finally i posted an issue ticket to bun github, you can find the ticket here https://github.com/oven-sh/bun/issues/12053

thing is it works nicely with nodejs and some old version of bun, gosh i can't remember what it was..
now just hoping a fix for the bun stack, since the other solution is only to write own http/1.1 fetcher using plain TCP sockets.
this is something i really wish not to do since its quite alot to code...

**Comment by 3zet** [comment]
Tried to fix it on my own but at the actual state of bun, bug is unfixable. 
```
call <> GetDetailedPlayerInfo
call <> GetServerPackMask
new socket created
PHPSESSID=77795d9868a5fb76a17ef2ed9cde9683
true
Dedimania: Authenticated.
call <> GetGameMode
call <> GetServerOptions
new socket created
PHPSESSID=77795d9868a5fb76a17ef2ed9cde9683
Dedimania: Updated players.
call <> GetGameMode
call <> GetServerOptions
new socket created
PHPSESSID=77795d9868a5fb76a17ef2ed9cde9683
Error in method_handler.php(903): Not Authenticated
```

**Comment by reaby** [comment]
fixed on 0.4.0 with change to use nodejs.
